src/memory/client/MemoryBrowser.tsx

```diff
@@ -318,4 +357,36 @@ export class MemoryBrowser extends React.Component<Props, State> {
       </div>
     );
   }
+
+  renderChildName() {
+    if (MemoryBrowser.firtTime) {
+      this.getAvailableChilds();
+      MemoryBrowser.firtTime = false;
+    }
+    const { childNames } = this.state;
+    let childNamesList =
+      childNames.length > 0 &&
+      childNames.map((item, i) => {
+        return (
+          <option key={i} value={item.id}>
+            {item.name}
+          </option>
+        );
+      }, this);
+    if (childNames.length > 0) {
+      return (
+        <div className="input-group">
+          <label>Child</label>
+          <select
+            defaultValue={this.childReq}
+            onChange={(event) => (this.childReq = event.target.value)}
```

Comment:
Please check the types here. As mentioned in the cdt-amalgamator change indexes should be numbers.

Reply:
I have updated it